base/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM azul/zulu-openjdk-debian:8
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
Is it by design that we want to pin this to latest and would we run into any issues down the line due to this?

Reply:
For new releases we do want to always get the latest version so that we get all the security updates.